dash-pipeline/bmv2/stages/trusted_vni.p4

```diff
@@ -0,0 +1,48 @@
+#ifndef _DASH_STAGE_TRUSTED_VNI_P4_
+#define _DASH_STAGE_TRUSTED_VNI_P4_
+
+control trusted_vni_stage(
+    inout headers_t hdr,
+    inout metadata_t meta)
+{
+    action permit() {}
+
+    action deny() {
```

Comment:
iirc, we have a global deny function and maybe we can use that instead.

Reply:
Use global deny function